### ISSUE TITLE
Issue with high color contrast of html table rendered by nbsphinx

### ISSUE THREAD

**Issue by andrey-jef** [lepture/shibuya#28]
Hi,

When parsing a `ipynb` file using `nbsphinx`, following [your example](https://shibuya.lepture.com/extensions/nbsphinx/), the rendered html table in the output cell in dark mode looks not visible due to high color contrast.

Is there any config to make it more readable? Thanks.

Here is a screencap in dark mode

![Screenshot 2024-02-26 060640](https://github.com/lepture/shibuya/assets/81280428/6dac44d1-e9da-43a4-b405-6ec2bc4c07b5)

Same page in light mode

![Screenshot 2024-02-26 060727](https://github.com/lepture/shibuya/assets/81280428/43b30fb3-5b55-4e33-8bc5-1e4ca8b256a9)

**Comment by lepture** [comment]
@andrey-jef can you send a PR to add the table in `https://github.com/lepture/shibuya/blob/master/docs/extensions/nbsphinx.ipynb`?

**Comment by lepture** [comment]
fixed by [fix: improve style for nbsphinx tables](https://github.com/lepture/shibuya/commit/ca8715ddc35da9e3b49efea053c7c0f445910a18)

**Comment by andrey-jef** [comment]
> @andrey-jef can you send a PR to add the table in `https://github.com/lepture/shibuya/blob/master/docs/extensions/nbsphinx.ipynb`?

Hi,

I saw that u already add the example of html data table. So I wont make a redundant PR 😄 

https://shibuya.lepture.com/extensions/nbsphinx/#Tables

**Comment by andrey-jef** [comment]
> fixed by [fix: improve style for nbsphinx tables](https://github.com/lepture/shibuya/commit/ca8715ddc35da9e3b49efea053c7c0f445910a18)

Did your recent fix for `nbsphinx` also works with `myst-nb`?

Last time I check, the same high color contrast happened with html tables rendered by `myst-nb`.

I'm curious with `myst-nb` since it allows [hide cell output content](https://myst-nb.readthedocs.io/en/latest/render/hiding.html). This makes the docs less cluttered vs every cell output was displayed as with `nbsphinx`.

**Comment by lepture** [comment]
Update shibuya theme.
